README.md

```diff
@@ -232,15 +222,23 @@ network stack. This is done in the following steps:
 Modify the command that runs the Image Caption Generator REST endpoint to map an additional port in the container to a
 port on the host machine. In the example below it is mapped to port `8088` on the host but other ports can also be used.
 
-    docker run -it -p 5000:5000 -p 8088:8088 --name max-im2txt max-im2txt
+    docker run -it -p 5000:5000 -p 8088:8088 --name max-image-caption-generator codait/max-image-caption-generator
 
 Build the web app image by running:
 
     docker build -t webapp .
 
 Run the web app container using:
 
-    docker run --net='container:max-im2txt' -it webapp
+    docker run --net='container:max-image-caption-generator' -it webapp
```

Comment:
Sorry for nitpicking some more, but really we should also change the tag `webapp` here -> `max-image-caption-generator-web-app` for consistency throughout

Reply:
good idea I'll update

Reply:
done